internal/alloycli/cmd_run.go

```diff
@@ -390,7 +390,7 @@ func (fr *alloyRun) Run(configPath string) error {
 }
 
 // getEnabledComponentsFunc returns a function that gets the current enabled components
-func getEnabledComponentsFunc(f *alloy.Alloy) func() map[string]interface{} {
+func getEnabledComponentsFunc(f *alloyRuntime.Alloy) func() map[string]interface{} {
```

Comment:
The struct being named Alloy (`alloyRuntime.Alloy` / `runtime.Alloy`, once the package path and imports are changed) feels a bit weird to me now that the package is named differently. Maybe it makes sense to rename the struct to `Runtime` so it's used as `runtime.Runtime`?

Reply:
Also fixed in HEAD :D